From: CompuLab <dev@compulab.co.il>
Subject: [PATCH] os-power-mode: Support EdgeAI-ORN families

Upstream-Status: Inappropriate [device configuration]
---
--- a/os-power-mode
+++ b/os-power-mode
@@ -89,13 +89,13 @@
     fi
 
     case $MACHINE in
-        jetson-agx-orin-devkit|jetson-agx-orin-devkit-64gb|jetson-agx-orin-industrial|jetson-orin-nx-seeed-j4012)
+        jetson-agx-orin-devkit|jetson-agx-orin-devkit-64gb|jetson-agx-orin-industrial|jetson-orin-nx-seeed-j4012|edgeai-orn-nx)
             power_modes=(["low"]="1" ["mid"]="3" ["high"]="0")
             ;;
         jetson-orin-nx-xavier-nx-devkit)
             power_modes=(["low"]="1" ["mid"]="4" ["high"]="0")
             ;;
-        jetson-orin-nano-devkit-nvme)
+        jetson-orin-nano-devkit-nvme|edgeai-orn-nano)
             power_modes=(["low"]="0" ["mid"]="1" ["high"]="2")
             ;;
         jetson-orin-nano-seeed-j3010)
