From d2f76707de32b947602c01c4ee1f9b0273e02cca Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 9 Mar 2026 08:38:41 +0200
Subject: [PATCH] compulab: Disable DecreaseRootfsRetryCount()

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
Upstream-Status: Pending
---
 Silicon/NVIDIA/Application/L4TLauncher/L4TRootfsValidation.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Silicon/NVIDIA/Application/L4TLauncher/L4TRootfsValidation.c b/Silicon/NVIDIA/Application/L4TLauncher/L4TRootfsValidation.c
index f12ab9f1..d145887a 100644
--- a/Silicon/NVIDIA/Application/L4TLauncher/L4TRootfsValidation.c
+++ b/Silicon/NVIDIA/Application/L4TLauncher/L4TRootfsValidation.c
@@ -585,7 +585,9 @@ DecreaseRootfsRetryCount (
 
   // The rootfs slot is bootable.
   if (RetryCount != 0) {
-    RetryCount--;
+    // Don't decrease the rootfs retry counter
+    // for production purpouse only
+    // RetryCount--;
     Status = SetRetryCountTomRootfsInfo (RootfsSlot, RetryCount);
     if (EFI_ERROR (Status)) {
       DEBUG ((
-- 
2.34.1

