From ea20aa54fa5e0e2e1798673897899df7ba489531 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Fri, 12 Dec 2025 12:44:53 +0200
Subject: [PATCH] compulab: dts: Remove unused padctl@3520000 fusb301@25

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
Upstream-Status: Pending
---
 ...ra234-p3768-0000+p3767-xxxx-nv-common.dtsi | 28 -------------------
 1 file changed, 28 deletions(-)

diff --git a/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi b/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi
index c70b517..8b27e64 100644
--- a/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi
+++ b/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi
@@ -178,18 +178,6 @@
 			};
 		};
 
-		padctl@3520000 {
-			ports {
-				usb2-0 {
-					port {
-						typec_p0: endpoint {
-							remote-endpoint = <&fusb_p0>;
-						};
-					};
-				};
-			};
-		};
-
 		i2c@c240000 {
 			status = "okay";
 			ina32211_1_40: ina3221@40 {
@@ -213,22 +201,6 @@
 					shunt-resistor-micro-ohms = <5000>;
 				};
 			};
-			fusb301@25 {
-				compatible = "onsemi,fusb301";
-				reg = <0x25>;
-				status = "okay";
-				#address-cells = <1>;
-				#size-cells = <0>;
-				interrupt-parent = <&gpio>;
-				interrupts = <TEGRA234_MAIN_GPIO(Z, 1) IRQ_TYPE_LEVEL_LOW>;
-				connector@0 {
-					port@0 {
-						fusb_p0: endpoint {
-							remote-endpoint = <&typec_p0>;
-						};
-					};
-				};
-			};
 		};
 
 		pcie-ep@14160000 {/* C4 - End Point */
-- 
2.34.1

