From 042d1c6641b15d4dcc298fe63e47a9f90d8ea518 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 9 Mar 2026 09:44:36 +0200
Subject: [PATCH 5/6] compulab: dts: Remove unused padctl@3520000 fusb301@25

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 ...ra234-p3768-0000+p3767-xxxx-nv-common.dtsi | 28 -------------------
 1 file changed, 28 deletions(-)

diff --git a/hardware/nvidia/t23x/nv-public/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi b/hardware/nvidia/t23x/nv-public/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi
index 2ed3600..f842be2 100644
--- a/hardware/nvidia/t23x/nv-public/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi
+++ b/hardware/nvidia/t23x/nv-public/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi
@@ -162,36 +162,8 @@
 			};
 		};
 
-		padctl@3520000 {
-			ports {
-				usb2-0 {
-					port {
-						typec_p0: endpoint {
-							remote-endpoint = <&fusb_p0>;
-						};
-					};
-				};
-			};
-		};
-
 		i2c@c240000 {
 			status = "okay";
-			fusb301@25 {
-				compatible = "onsemi,fusb301";
-				reg = <0x25>;
-				status = "okay";
-				#address-cells = <1>;
-				#size-cells = <0>;
-				interrupt-parent = <&gpio>;
-				interrupts = <TEGRA234_MAIN_GPIO(Z, 1) IRQ_TYPE_LEVEL_LOW>;
-				connector@0 {
-					port@0 {
-						fusb_p0: endpoint {
-							remote-endpoint = <&typec_p0>;
-						};
-					};
-				};
-			};
 		};
 
 		/* PWM1, 40pin header, pin 15 */
-- 
2.34.1

